app/views/items/show.html.erb

```diff
@@ -15,5 +15,15 @@
       Item Retired
```

Comment:
There's a lot of code in this view I think we're going to want to get rid of. The `item.active?` and `add to cart` functionality we're probably not going to use.